.gitlab-ci.yml

```diff
@@ -0,0 +1,51 @@
+gitlab-dev:
+  tags:
+    - meao
+    - gcp
+  only:
+    - gitlab
+  variables:
+    NAMESPACE: nucleus-dev
+  script:
+    - docker/bin/build_images.sh
+    - docker/bin/push2dockerhub.sh
+    - bin/update-config.sh
+
+master:
+  tags:
+    - meao
+    - gcp
+  only:
+    - master
+  variables:
+    NAMESPACE: nucleus-dev
+  script:
+    - docker/bin/build_images.sh
+    - docker/bin/push2dockerhub.sh
+    - bin/update-config.sh
```

Comment:
Is there a reason we can't have these two combined (adding an entry in `only`)? Or are they just separated for now to call out that one is testing and will be deleted later?

Reply:
as discussed on zoom, let's keep both jobs for now, and we can use the gitlab branch in the furture for testing new versions of .gitlab-ci.yml outside of the master branch.